electron/main/index.ts

```diff
@@ -182,6 +222,19 @@ async function createWindow() {
   ipcMain.handle('getChain', (event, coin) => {
     return sdk.coin(coin).getChain();
   });
+
+  ipcMain.handle('getUser', () => {
+    try {
+      return sdk.me();
+    } catch (e: unknown) {
+      return new Error(handleSdkError(e));
+    }
+  });
+
+  ipcMain.handle('isSdkAuthenticated', async event => {
+    const sdkJson = sdk.toJSON();
+    return Promise.resolve(sdkJson.user || sdkJson.token);
```

Comment:
```suggestion
    return Promise.resolve(sdkJson.user ?? sdkJson.token);
```
I'm assuming the intent was to fall back on `token` if `user` was `null` and not `false`

Reply:
The function is returning a boolean, `||` should be correct here.